packages/wrangler/src/delete.ts

```diff
@@ -98,3 +115,130 @@ async function deleteSiteNamespaceIfExisting(
 		logger.log(`🌀 Deleted asset namespace for Workers Site "${ns.title}"`);
 	}
 }
+
+type ServiceReference = {
+	name: string;
+	service: string;
+	environment: string;
+};
+
+type DurableObjectServiceReference = ServiceReference & {
+	durable_object_namespace_id: string;
+	durable_object_namespace_name: string;
+};
+
+type DispatchOutboundsServiceReference = ServiceReference & {
+	namespace: string;
+	params: { name: string }[];
+};
```

Comment:
Can this reuse types from https://github.com/cloudflare/workers-sdk/blob/a9349a89296dc98ac8fc52dbb013f298c9596d8f/packages/wrangler/src/worker.ts?

Reply:
moved types to the shared `worker.ts`